bridge-proxy/src/bridge-proxy.rs

```diff
@@ -83,4 +88,23 @@ pub trait BridgeProxyContract:
         self.pending_transactions().get_or_else(tx_id, || panic!("Invalid tx id"))
     }
 
+    #[view(getPendingTransactions)]
+    fn get_pending_transactions(&self) -> MultiValueEncoded<MultiValue2<usize, EthTransaction<Self::Api>>> {
+        let lowest_tx_id = self.lowest_tx_id().get();
+        let len = self.pending_transactions().len();
+        
+        let mut transactions = MultiValueEncoded::new();
+        for i in lowest_tx_id..=len {
```

Comment:
I think the following statement would be more efficient gas wise:
` for (i, transaction) in self.pending_transactions().iter().enumerate()`.

It would be less storage accesses.